mlos_core/mlos_core/optimizers/bayesian_optimizers/smac_optimizer.py

```diff
@@ -241,13 +257,14 @@ def _register(self, configurations: pd.DataFrame, scores: pd.Series, context: Op
         from smac.runhistory import StatusType, TrialInfo, TrialValue  # pylint: disable=import-outside-toplevel
 
         if context is not None:
-            raise NotImplementedError()
+            warn(f"Not Implemented: Ignoring context {list(context.columns)}", UserWarning)
```

Comment:
Maybe leave that change for a separate PR?

Reply:
makes sense! smaller PRs are always good. I will do it now